Sources/RootsEncoder/main.swift

```diff
@@ -98,13 +98,9 @@ if CommandLine.arguments.contains("test") {
   s += "import Foundation\n"
   s += "\n"
 
-  s += "public func roots_pem() -> String {\n"
+  s += "public let roots_pem: String = \"\"\"\n"
```

Comment:
Should we make this a constant using the `k` prefix (i.e., `kRootsPem`)?

Reply:
Great idea! I've renamed the constant to `kRootCertificates` now; what do you think of that name?

Reply:
Yea sounds great!